app/models/optimized_image.rb

```diff
@@ -49,7 +49,8 @@ def self.create_for(upload, width, height, opts = {})
     end
 
     # prefer to look up the thumbnail without grabbing any locks
-    thumbnail = find_by(upload_id: upload.id, width: width, height: height)
+    extension = ".#{opts[:format] || upload.extension}"
```

Comment:
Do we want to store the leading `.` for every single optimized images? 🤔

Reply:
Is "format" to correct name? Why not "extension"?

Reply:
agree this needs cleaning up, but I worry about the huge rabbit hole here. The . and calling this format is very confusing.